packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -108,6 +108,13 @@ import {
   S3BucketCard,
   LambdaFunctionCard,
   IAMUserCard,
+  IAMRoleCard,
```

Comment:
@sblausten this adds an iam role card to show detailsa bout the iam role if an annotation is provider `amazon.com/aws-cluster-arn`